book/005-asynchronous-tooling.md

```diff
@@ -274,23 +274,23 @@ fetch('/some-endpoint/users')
   );
 ```
 
-There's nothing wrong with either approach, just make sure you and your team agrees on which method is preferred.
+There's nothing wrong with either approach, just make sure you and your team agree on which method is preferred.
 
-**IMPORTANT** As of Node v10, Unhandled rejected promises will halt and/or crash the Node process. If you aren't handling rejected promises now, you will need to it soon, otherwise your whole program will crash.
+**IMPORTANT** As of Node v10, Unhandled rejected promises will halt and/or crash the Node process. If you aren't handling rejected promises now, you will need to soon, otherwise your whole program will crash.
```

Comment:
Yikes. It must have been a late night while writing this passage...

Probably best to split this into 2 sentences.

`... rejected promises now, you will need to soon. Otherwise, the Node process will crash`